node-notes-cli/app.js

```diff
@@ -0,0 +1,74 @@
+import * as fs from 'node:fs/promises';
+
+async function readNotes() {
+  try {
+    const fileContent = await fs.readFile('data.json', 'utf-8');
```

Comment:
You repeat these two lines of code several times. You might want to encapsulate them into a function.